src/components/Container/Container.styled.ts

```diff
@@ -0,0 +1,6 @@
+import styled from 'styled-components';
+
+export const StyledContainer = styled.div`
+  width: 100vw;
+  margin: 40px;
```

Comment:
it's ok, but there is an approach to not use margin on elements but add a gap on the parent. to take this in mind in the future.